src/components/Table/Table.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+
+type Props = {
+  people: Person[];
+};
+
+export const Table: React.FC<Props> = ({ people }) => {
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map(person => (
+          <PersonLink key={person.slug} person={person} />
```

Comment:
It looks like you are trying to render a list of `PersonLink` components inside a table body, but `PersonLink` seems to be a single table row (`tr`). This will result in invalid HTML because the `tr` elements must be direct children of `tbody`, `thead`, or `tfoot`. You should directly return a `tr` from the map function and place the `PersonLink` component inside of it.